features/authentication.feature

```diff
@@ -95,7 +95,7 @@ Feature: OAuth authentication
       Duplicate value for "description"
       """
     And the exit status should be 1
-    And the file "../home/.config/hub" should not exist
+    And the file "../home/.config/hub" should contain ""
```

Comment:
I agree. Updated. Also updated new tests to check that file is not present after the test.